Python/Search/BinarySearch.py

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Add the licence header. (https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md)